packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -1079,6 +1079,46 @@ class AlignTransition extends AnimatedWidget {
 /// Animated version of a [DefaultTextStyle] that animates the different properties
 /// of its [TextStyle].
 ///
+/// {@tool dartpad --template=stateful_widget_material_ticker}
+///
+/// The following code implements the [DefaultTextStyleTransition]
```

Comment:
nit: sentences should end with `.`

Reply:
@goderbauer I added `.` while I adding an explanation of the example.